### PR TITLE
Fixed showing adherent registration date

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: templates/adherent/overview.html.twig [hunk 1]
```diff
@@ -8,7 +8,11 @@
         <span class="settings__username">{{ app.user.fullname }}</span>
         <span class="text--gray">({{ app.user.emailaddress }})</span>.
     </div>
-    <div class="settings__membership">Adhérent{{ app.user.gender == "female" ? "e" }} depuis {{ app.user.activatedAt|localizeddate('none', 'none', 'fr_FR', null, 'MMMM Y') }}.</div>
+    {% if app.user.activatedAt %}
+        <div class="settings__membership">Adhérent{{ app.user.gender == "female" ? "e" }} depuis {{ app.user.registeredAt|localizeddate('none', 'none', 'fr_FR', null, 'MMMM Y') }}.</div>
```
Comment (Remg) [inline templates/adherent/overview.html.twig]: `{{ app.user.isFemale ? "e" }}`

File: tests/Controller/ControllerTestTrait.php [hunk 2]
```diff
@@ -62,7 +62,7 @@ public function authenticateAsAdherent(Client $client, string $emailAddress, str
         $shouldBeRedirectedTo = 'http://'.Config::APP_HOST.'/evenements';
 
         if ($shouldBeRedirectedTo !== $client->getResponse()->headers->get('location')) {
-            throw new \RuntimeException(
+            $this->fail(
```
Comment (Remg) [inline tests/Controller/ControllerTestTrait.php]: :+1:

File: templates/adherent/overview.html.twig (hunk 1 above)
Comment (HeahDude) [reply templates/adherent/overview.html.twig]: Not related, but why not?

Comment (HeahDude) [reply templates/adherent/overview.html.twig]: Done.

**Pull request merged**